src/main/java/io/fabric8/maven/docker/util/AnsiLogger.java

```diff
@@ -106,7 +112,7 @@ public boolean isDebugEnabled() {
     }
 
     public boolean isVerboseEnabled() {
-        return verbose;
+        return !verbose.isEmpty();
```

Comment:
I think if you use -Ddocker.verbose=false then verbose logging should be disabled as well. Please add a check for "false", too.